torch/csrc/jit/mobile/import.cpp

```diff
@@ -150,15 +150,15 @@ c10::IValue BytecodeDeserializer::readArchive(const std::string& archive_name) {
   };
 
   auto class_resolver = [&](const c10::QualifiedName& qn) {
-    if (compilation_unit_->get_class(qn) == nullptr) {
-      auto typeptr = ClassType::create(qn, compilation_unit_, true);
-      compilation_unit_->register_type(typeptr);
-    }
-    return c10::StrongTypePtr(
-        compilation_unit_, compilation_unit_->get_class(qn));
+    return nullptr;
   };
 
-  auto obj_loader = [&](at::StrongTypePtr type, IValue input) {
+  auto obj_loader = [&](c10::ClassTypePtr typePtr, IValue input) {
```

Comment:
Probaby don't want to create a new class type for every object. It would be better to a have a single class type for everything so there are not a lot of different "dummy" types floating around.